core-web/libs/block-editor/src/lib/shared/components/suggestions/suggestions.component.html

```diff
@@ -4,7 +4,7 @@
             <h3>{{ title }}</h3>
         }
         <dot-suggestion-list #list>
-            @for (item of items; track $index) {
+            @for (item of items; track item) {
```

Comment:
Is it okay to `track` an `object`? Should we track the `item.id`?

Reply:
@rjvelazco The error was for don't use an obj and in this case this item does not have an id.